src/main/java/world/bentobox/bentobox/api/commands/admin/AdminSetSpawnPointCommand.java

```diff
@@ -0,0 +1,110 @@
+//
+// Created by BONNe
+// Copyright - 2020
+//
+
+
+package world.bentobox.bentobox.api.commands.admin;
```

Comment:
```suggestion
package world.bentobox.bentobox.api.commands.admin;
```

"copyright" information is already held in the Javadoc